frontend/src/features/admin-form/responses/IndividualResponsePage/IndividualResponseNavbar.tsx

```diff
@@ -138,13 +141,13 @@ export const IndividualResponseNavbar = (): JSX.Element => {
           isDisabled={!prevSubmissionId || isAnyFetching}
           onClick={handleNavigatePrev}
           icon={<BiChevronLeft />}
-          aria-label="Previous submission"
+          aria-label={`${t('features.common.previous')} ${t('features.common.submission')}`}
         />
         <IconButton
           isDisabled={!nextSubmissionId || isAnyFetching}
           onClick={handleNavigateNext}
           icon={<BiChevronRight />}
-          aria-label="Next submission"
+          aria-label={`${t('features.common.next')} ${t('features.common.submission')}`}
```

Comment:
I get that it makes sense to abstract common terms, but there exist languages where subject, adjective, verb word-order doesn't follow the same structure as English. I.e., In Bahasa, `next steps` is expressed as `steps next`.

Generally, we try to codify them as complete intents, for instance this button is to "download more", then we should codify the full intent of "download more" rather than "download", and "more".

Reply:
Ah, I missed this. Let me deal with it now.